app/.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
```

Comment:
- [ ] This `.github/workflows` folder shouldn't be here in this folder.  This one has the correct linters.yml file.  Kindly move it to the root of your project. 👍